### PR TITLE
Update RKE CIS Benchmark 1.5 and 1.6 hardening guide and verification guide

### PR DESCRIPTION
**Pull request by dhruvmewada15**
Update RKE CIS Benchmark 1.5 and 1.6 hardening guide and verification guide for CIS profiles: 

- [rke-profile-hardened-1.5](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.5-hardened)
- [rke-profile-hardened-1.6](https://github.com/rancher/security-scan/tree/master/package/cfg/rke-cis-1.6-hardened)

The pages that were updated are:

[ rke-profile-hardened-1.5 CIS Hardening Guide](https://rancher.com/docs/rancher/v2.5/en/security/rancher-2.5/1.5-hardening-2.5/)
[ rke-profile-hardened-1.5 CIS Self Assessment Guide](https://rancher.com/docs/rancher/v2.5/en/security/rancher-2.5/1.5-benchmark-2.5/)

[ rke-profile-hardened-1.6 CIS Hardening Guide](https://rancher.com/docs/rancher/v2.5/en/security/rancher-2.5/1.6-hardening-2.5/)
[ rke-profile-hardened-1.6 CIS Self Assessment Guide](https://rancher.com/docs/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/)

### CODE REVIEW

**Comment by dhruvmewada15** [comment]
Screenshot of changes in rke-profile-hardened-1.5 as tested on local server:
image
image
image

**Comment by brandond** [comment]
This looks OK to me but since it's for RKE not RKE2/K3s I shouldn't be an approver.

**Review by macedogm — COMMENTED** [review]
The flag `bind-address: 127.0.0.1` added to the hardening pages looks good to me.

I'm just not sure about the updates done in the assessment docs. Were they done manually or after executing the CIS scan and generating those automatically with the scripts in https://github.com/rancher/docs/tree/master/scripts/converters ?

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md [hunk 1]
```diff
@@ -1803,13 +1803,13 @@ on the master node and ensure the correct value for the --bind-address parameter
...
+root        4788    4773  4 16:16 ?        00:00:09 kube-controller-manager --configure-cloud-routes=false --cloud-provider= --service-cluster-ip-range=10.43.0.0/16 --v=2 --bind-address=127.0.0.1 --pod-eviction-timeout=5m0s --leader-elect=true --cluster-cidr=10.42.0.0/16 --kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-controller-manager.yaml --profiling=false --node-monitor-grace-period=40s --service-account-private-key-file=/etc/kubernetes/ssl/kube-service-account-token-key.pem --address=0.0.0.0 --allow-untagged-cloud=true --root-ca-file=/etc/kubernetes/ssl/kube-ca.pem --allocate-node-cidrs=true --enable-hostpath-provisioner=false --terminated-pod-gc-threshold=1000 --feature-gates=RotateKubeletServerCertificate=true --use-service-account-credentials=true
```
Comment (macedogm) [inline content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md:1812]: Isn't this a conflicting configuration `--bind-address=127.0.0.1` and `--address=0.0.0.0`?

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md [hunk 2]
```diff
@@ -1859,13 +1859,13 @@ on the master node and ensure the correct value for the --bind-address parameter
...
 ```console
-'--bind-address' is not present OR '--bind-address' is not present
+'--bind-address' argument is set to 127.0.0.1
 ```
 
 **Returned Value**:
 
 ```console
-root        4947    4930  1 16:16 ?        00:00:02 kube-scheduler --kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-scheduler.yaml --leader-elect=true --profiling=false --v=2 --address=0.0.0.0
+root        4947    4930  1 16:16 ?        00:00:02 kube-scheduler --kubeconfig=/etc/kubernetes/ssl/kubecfg-kube-scheduler.yaml --leader-elect=true --profiling=false --v=2 --address=0.0.0.0 --bind-address=127.0.0.1
```
Comment (macedogm) [inline content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md:1868]: Same as the above comment.

Isn't this a conflicting configuration `--bind-address=127.0.0.1` and `--address=0.0.0.0`?

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md (hunk 1 above)
Comment (dhruvmewada15) [reply content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md:1812]: --address flag is DEPRECTAED and we are not using it in our RKE Template config. By default it sets the value to `0.0.0.0` for address flag

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md (hunk 2 above)
Comment (dhruvmewada15) [reply content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md:1868]: --address flag is DEPRECTAED and we are not using it in our RKE Template config. By default it sets the value to `0.0.0.0` for address flag

Comment (macedogm) [reply content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md:1868]: See comment in above thread.

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md (hunk 1 above)
Comment (dhruvmewada15) [reply content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md]: @macedogm We have made the changes as suggested. Please review them if there is any feedback

File: content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md (hunk 2 above)
Comment (dhruvmewada15) [reply content/rancher/v2.5/en/security/rancher-2.5/1.6-benchmark-2.5/_index.md]: @macedogm Kindly refer to the comment [here](https://github.com/rancher/docs/pull/3902#discussion_r828055219)

**Review by macedogm — APPROVED** [review]
LGTM

**Review by jtravee — APPROVED** [review]

**Review by prachidamle — COMMENTED** [review]
LGTM

**Pull request merged**